kovid.gemspec

```diff
@@ -8,8 +8,9 @@ Gem::Specification.new do |spec|
   spec.authors       = ['Emmanuel Hayford']
   spec.email         = ['siawmensah@gmail.com']
 
-  spec.summary       = 'A CLI to fetch and compare the 2019 coronavirus pandemic statistics.'
-  spec.description   = 'A CLI to fetch and compare the 2019 coronavirus pandemic statistics.'
+  smry = 'A CLI to fetch and compare the 2019 coronavirus pandemic statistics.'
```

Comment:
Instead of `smry` can it be `description` or  `summary`? `smry` doesn't convey meaning IMO.

Reply:
I used the abbreviated name to avoid passing the 80 column limit. But we can break the string in two lines to solve this, so I'll rename the variable to `summary`. :+1:

Reply:
Also please feel free to modify the cop rules at will. If it's complaining your string is 25 chars long, and you have 23 chars, change it to accept 25 chars! 😄after all Rubocop is only a guide.

Reply:
I'll try to stick to the default rules, but will keep in mind modifying them if it gets too tough.
Thanks for the feedback. xD